README.md

```diff
@@ -43,18 +43,106 @@ Most methods pass values as-is into HTTP requests. These can be populated using
 const { RestClient } = require('bitmart-api');
 
 const client = new RestClient({
-  apiKey: "yourAPIKeyHere",
-  apiSecret: "yourAPISecretHere",
-  apiMemo: "yourAPIMemoHere
+  apiKey: 'yourAPIKeyHere',
+  apiSecret: 'yourAPISecretHere',
+  apiMemo: 'yourAPIMemoHere',
 });
 
-client.getAccountBalancesV1()
-.then((result) => {
-  console.log('getAccountBalancesV1 result: ', result);
-})
-.catch((err) => {
-  console.error('getAccountBalancesV1 error: ', err);
+client
+  .getAccountBalancesV1()
+  .then((result) => {
+    console.log('getAccountBalancesV1 result: ', result);
+  })
+  .catch((err) => {
+    console.error('getAccountBalancesV1 error: ', err);
+  });
+```
+
+### WebSocket
+
+- All available WebSockets can be used via a shared WebsocketClient
+- Simple instructions are below, for more examples check [examples](./examples)
+
+```typescript
+// create websocket client
+// If public client, doesn't need API keys
+const client = new WebsocketClient();
+
+// If private client, needs API keys
```

Comment:
Can you add a note here that custom logger is optional (or remove the custom logger, since there's an example that shows how to customise logging)